doc-output/configure-the-report-engine/typereferences-element.md

```diff
@@ -17,11 +17,11 @@ table th:nth-of-type(2) {
 }
 </style>
 
-# TypeReferences Element Overview
+# typeReferences Element Overview
```

Comment:
@todorarabadzhiev Can you please check if the **typeReferences** element in configuration file is case-insensitive?

Reply:
It is case-sensitive, which will be corrected for the next release.